package.json

```diff
@@ -130,6 +130,7 @@
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-processor-styled-components": "^1.8.0",
     "testcafe-reporter-xunit": "^2.1.0",
+    "uuid": "^3.3.2",
```

Comment:
Can we just use a local sequence generator to avoid another dependency?

Eg:
```js
let count = 0;
function getId(): string {
  return `id-${count++}`;
}
```

Reply:
I should be a cryptographically save random number but to prove a test i will change it

Reply:
This looks better i think

`let count = 0;
function getNonce(): string {
  return `ThisShouldBeACryptographicallySecurePseudoRandomNumber-${count++}`;
}`